spec/requests/api/moves_controller_role_access_rswag_spec.rb

```diff
@@ -23,15 +23,18 @@
     let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{token.token}") }
     let(:schema) { load_yaml_schema('get_moves_responses.yaml') }
 
-    let!(:pentonville) { create :location, :with_moves, suppliers: [pentonville_supplier] }
-    let!(:birmingham) do
-      create :location,
-             :with_moves,
-             key: 'hmp_birmingham',
-             title: 'HMP Birmingham',
-             nomis_agency_id: 'BMI',
-             suppliers: [birmingham_supplier]
-    end
+    # let!(:pentonville) { create :location, :with_moves, suppliers: [pentonville_supplier] }
+
+    let!(:moves) { create_list(:move, 2, supplier: pentonville_supplier) }
```

Comment:
We need to make sure we're creating moves for the birmingham supplier:

```ruby
    let!(:filtered_out_moves) { create_list(:move, 2, supplier: birmingham_supplier) }
```

Although the name birmingham_supplier implies 1-2-1 relationship between a location and a supplier which of course isn't correct.